components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -26,6 +29,8 @@ const useStyles = makeStyles((theme: Theme) =>
             },
         },
         listItem: {
+            color: theme.palette.text.secondary,
```

Comment:
Does this do anything? I set it to red and nothing changed.

Reply:
No they don't. I am taking them out. Ideally, these shall be passed into `InfoListItem` via a `classes` prop. Right now the font color, the font weight, etc, are all controlled by other props.